src/main.ts

```diff
@@ -0,0 +1,161 @@
+import * as core from '@actions/core';
+import * as github from '@actions/github';
+import * as fs from 'fs';
+import { request } from '@octokit/request';
+import { Octokit } from "@octokit/core";
+fs.readdir('./', (err, files) => {
+  if (err)
+    console.log(err);
+  else {
+
+    //Check for example and Contribution in README
+    readmeChecks(files);
+
+    //Check for CODEOWNERS file in .github folder
+    codeownerCheck();
```

Comment:
Follow camelCase convention for naming variables/function names
example : this can be codeOwnerCheck

follow the same for other as well